packages/tasit-contracts/test/EstateRegistry.js

```diff
@@ -1,24 +1,31 @@
-const abi = require("../3rd-parties/decentraland/land/build/contracts/EstateRegistry.json")
-  .abi;
-const localAddresses = require("../3rd-parties/decentraland/addresses").local;
-const { LANDRegistry: landAddress, EstateRegistry: address } = localAddresses;
+const TasitContracts = require("../dist/TasitContracts").TasitContracts;
```

Comment:
Is this line worth splitting out into two lines with destructuring assignment like we do elsewhere, or is this syntax better when we're doing `require`s?

Reply:
I think that's fine with `require()`

Reply:
Okay thanks, resolving this